packages/core/src/components/Form/Checkbox.stories.js

```diff
@@ -20,25 +22,36 @@ const Checkbox = ({
   const hasError = state === 'error';
   const checkedAttr = checked ? 'checked' : '';
 
+  const storyId = `use-a-more-meaningful-id__d:${
+    disabled.length > 0 ? 'true' : 'false'
+  }__e:${hasError}__c:${checked.toString()}`;
+  console.dir(storyId);
```

Comment:
NIT: did you mean to leave this in?

Reply:
Eek! Great catch—thank you.

Reply:
(The ID value is intentional—but there may be a better way to convey the intent.)